example/HelloWorldFunction/pom.xml

```diff
@@ -80,9 +80,9 @@
     <build>
       <plugins>
         <plugin>
-             <groupId>org.codehaus.mojo</groupId>
+             <groupId>com.nickwongdev</groupId>
```

Comment:
Maybe add a comment on why this fork from codehaus aspectj-maven-plugin is needed? Something like
<!--
                 mojohaus aspectj-maven-plugin support for Java 11 is being handled in
                 https://github.com/mojohaus/aspectj-maven-plugin/pull/45
                 Here using a project fork supporting Java 11, 12 and 13
             -->

Reply:
Referred the issue itself.